app/src/globals/interfaces.ts

```diff
@@ -0,0 +1,16 @@
+/* Global Interfaces */
+
+//Asset Types
+export interface AssetTypeData { //TODO Finish this
```

Comment:
Think it might be worth exporting the interface and then importing it when needed?

Wary of globs like this in ts

Reply:
Thought that’s what this did? These interfaces were moved to this file as were used in a number of asset pages and was just duplicating code